models/issue.go

```diff
@@ -1210,7 +1210,7 @@ func (opts *IssuesOptions) setupSession(sess *xorm.Session) {
 	}
 
 	if opts.IsArchived != util.OptionalBoolNone {
-		sess.Join("INNER", "repository", "issue.repo_id = repository.id").And(builder.Eq{"repository.is_archived": opts.IsArchived.IsTrue()})
```

Comment:
CI fail as `repository` for `repository.is_archived` is not yet defined (due to it being defined in the join). This conditional needs to be below where the join is set